### DISCUSSION TITLE
Unicode 16: Symbols for Legacy Computing

### DISCUSSION THREAD

**Discussion by jayaddison** [Textualize/textual#4983] [General]
This isn't a particular idea / question /suggestion, but I thought that some people here might be interested in the [Symbols for Legacy Computing](https://www.unicode.org/charts/PDF/Unicode-16.0/U160-1FB00.pdf) character codes that are include in the [recently-announced Unicode 16.0 Standard](http://blog.unicode.org/2024/09/announcing-unicode-standard-version-160.html).

Some of them appear useful for creating user-interface elements, and some of them appear useful to compose somewhat teletext-like graphical images.

**Comment by TomJGooding** [comment]
I've actually been looking at using one of these characters for the [`hatch` style](https://textual.textualize.io/styles/hatch/) in my app. My only concern is how widely these characters are supported across different fonts.

**Comment by fancidev** [comment]
I wanted to some of the block characters for chart plotting, but they are not supported by the font of my Windows Terminal :-(
